### ISSUE TITLE
Questions about training details 

### ISSUE THREAD

**Issue by Solacex** [zehuichen123/AutoAlignV2#2]
First, thanks for your inspiring work! I am trying to reproduce the reported results with this report and have following questions. 

1, Could you provide the pretrained model of 2D detectors? I cannot find it in the mmdetection3D.
2, The current config files seem to make some simplification on the training, including using 1/8 dataset, 9 sweeps and 1 images per scene. So is there any difference with the full-set training on the  lr configs? 
3, Which fusion layer is the final version? multi_voxel_deform_fusion v2/v3 fast  .etc? 

Thanks in advance.

**Comment by zehuichen123** [comment]
Thanks for your attention. 
1. The 2D pretrained weight is provided here (https://drive.google.com/file/d/1DuA16ocF8Dmfl0nC3NNSKG_Yqq-aXuLH/view?usp=sharing), I will add them to the readme later.
2. No, all the settings are the same.
3. multi_voxel_deformfusion.py is the final version.

**Comment by Solacex** [comment]
Thanks for your prompt reply! It really helps.

**Comment by zehuichen123** [comment]
@Solacex  If you want to reproduce the score on the leaderboard, you may need to set the image input size to 800x9999(short side to 800, I will upload this pretrain model later). And during our experiments, we find directly using the front view image as the training input only is comparable with image-level dropout of 3 images.

**Comment by Solacex** [comment]
Thanks for your suggestion!

By far, on the 1/8 dataset, the final result can reach 57.4.

**Comment by Solacex** [comment]
BTW, we also observe the using 6 images does not lead to large improvement than using one.

**Comment by zehuichen123** [comment]
> Thanks for your suggestion!
> 
> By far, on the 1/8 dataset, the final result can reach 57.4.

Is this result tested with TTA? My results with TTA is: pts_bbox_NuScenes/NDS: 0.6307, pts_bbox_NuScenes/mAP: 0.5846.

**Comment by Solacex** [comment]
I will try the higher resolution soon. The reported result is with the lower resolution in the current config.

**Comment by zehuichen123** [comment]
> I will try the higher resolution soon. The reported result is with the lower resolution in the current config.

That's wired. This 1/8 result is obtained with a 640x9999 image backbone. I will recheck the code today.

**Comment by Solacex** [comment]
In the final version, how do you set the ''light'' flag in the DeformTransLayer? I tried to activate it, but it requires too large memories.

**Comment by zehuichen123** [comment]
> In the final version, how do you set the ''light'' flag in the DeformTransLayer? I tried to activate it, but it requires too large memories.

We keep the default setting for this arg (light = True) in the final version.

**Comment by Solacex** [comment]
What is the result of the pretrained 3D backbone? I have run with 9 or 10 sweeps, and the results are 49.9 and 50.5 mAP, respectively. [using centerpoint_voxel_nus_8subset_dv_aug.py]

**Comment by zehuichen123** [comment]
I've uploaded the final config version and you may refer to that for the experimental setting. I can not find this centerpoint_voxel_nus_8subset_dv_aug experiments in my machine. :(

**Comment by Solacex** [comment]
Thanks, I will try it soon.

**Comment by Solacex** [comment]
Hey, can you report the results of the first stage [lidar only] with the full set? 
Thanks in advance.

**Comment by zehuichen123** [comment]
Hi, the LiDAR only model is:
pts_bbox_NuScenes/NDS: 0.6884, pts_bbox_NuScenes/mAP: 0.6256
with TTA (only x,y flip)

**Comment by Solacex** [comment]
Okay, how many GPUs do you use for the training, 4x4 or 4x8? 
BTW, would you mind discussing some further details with me on WeChat (Li_Guangrui)?

**Comment by zehuichen123** [comment]
8 GPUS by default. Sure.

**Comment by vehxianfish** [comment]
@Solacex,Has the code been released? I don't see the code released.

**Comment by vehxianfish** [comment]
> @vehxianfish Thx for your attention. You may refer to some forks of this codebase (but they may not be the latest). The code will release once getting the approval from sensetime.

thanks!
